.github/workflows/build_and_test_release_latest.yml

```diff
@@ -0,0 +1,59 @@
+name: Build & Test release-latest
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every hour. This helps detect flakiness,
+    # and broken external dependencies.
+    - cron:  '0 * * * *'
+
+jobs:
+  build_and_test_release_latest:
+    name: Build and Test Release Latest ROS ${{ matrix.ros_version }} ${{ matrix.ros_distro }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro: [kinetic, melodic, dashing]
+        include:
+        - ros_distro: kinetic
+          ubuntu_distro: xenial
+          ros_version: 1
+          extra_cmake_args: --cmake-target tests
+        - ros_distro: melodic
+          ubuntu_distro: bionic
+          ros_version: 1
+          extra_cmake_args: --cmake-target tests
+        - ros_distro: dashing
+          ubuntu_distro: bionic
+          ros_version: 2
+          extra_cmake_args: ""
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.ros_distro }}-ros-base-latest
+    outputs:
+      build_result: ${{ steps.build.outcome }}
```

Comment:
I don't think this `outputs` is doing anything, there is no step with `name: build` (or `test`), and the named outputs `build_result` and `test_result` aren't being read anywhere.

Reply:
This is leftover from the  log to CloudWatch action (currently removed as we haven't added the secrets to this repo yet).

Reply:
does that portion actually use these variables, or is it dangling from an older rendition of the reporting code? E.g. https://github.com/aws-robotics/rosbag-uploader-ros1/blob/master/.github/workflows/main.yml does not use `build_result` anywhere